lib/matplotlib/_mathtext.py

```diff
@@ -2160,6 +2160,8 @@ def accent(self, s, loc, toks):
             accent_box.shrink()
             accent_box.shrink()
         centered = HCentered([Hbox(sym.width / 4.0), accent_box])
+        while isinstance(sym, Hlist):
+            sym = sym.children[0]
```

Comment:
I'm probably wrong, but shouldn't line 2162 go below the two newly added lines? The way it currently is, `sym` is a `Char` in case of `\dot s` and an `Hlist` with kerning in case of `\dot{s}`, whereby `centered` will be different in those two cases.